README.md

```diff
@@ -52,25 +62,29 @@ cd one-app
 npm ci --no-optional
 ```
 
-**Build a Module:**
+**Serve your Module to One App**
+
+At the root of your `one-app` repo, run:
 
 ```bash
-cd prod-sample/sample-modules/frank-lloyd-root/0.0.0
-npm ci
-cd ../../../..
-npm run serve-module prod-sample/sample-modules/frank-lloyd-root/0.0.0
-# this is used by the frank-lloyd-root module
-export ONE_CONFIG_ENV=development
+npm run serve-module <local-path-to-generated-module>
+# e.g. npm run serve-module ../my-first-module
 ```
 
-**Declare a Root Module and Start One App:**
+The `serve-module` command generates a `static` folder in the `one-app` root directory, containing a `module-map.json` and a `modules` folder with your bundled module code. Paired with the built-in [one-app-dev-cdn](https://github.com/americanexpress/one-app-dev-cdn) library, you're able to utilize the [Holocron Module Map](#-building-and-deploying-a-holocron-module-map) while running your entire One App instance locally. No need to deploy and fetch remote assets from a CDN at this step.
```

Comment:
I would be super explicit and show the a code block output of a `tree` command (`brew install tree`) for the `static` folder to demonstrate the contents and location of the `static` folder.

Reply:
great idea. Added in 5d97705